apps/spruce/src/components/Table/BaseTable.tsx

```diff
@@ -220,65 +255,51 @@ const RenderableRow = <T extends LGRowData>({
 }: {
   dataCyRow?: string;
   row: LeafyGreenTableRow<T>;
-  virtualRow: VirtualItem;
+  virtualRow?: VirtualItem;
   isSelected?: boolean;
 }) => (
-  <Row
-    className={css`
-      &[aria-hidden="false"] td > div {
-        max-height: unset;
-      }
-      ${isSelected &&
-      `
-        background-color: ${blue.light3} !important;
-        font-weight:bold;
+  <Fragment key={row.id}>
+    {!row.isExpandedContent && (
+      <Row
+        className={css`
+          ${isSelected &&
+          `
+           background-color: ${blue.light3} !important;
+           font-weight:bold;
+           `}
         `}
-    `}
-    data-cy={dataCyRow}
-    data-selected={isSelected}
-    row={row}
-    virtualRow={virtualRow}
-  >
-    {row.getVisibleCells().map((cell) => (
-      <Cell key={cell.id} style={cellPaddingStyle}>
-        {flexRender(cell.column.columnDef.cell, cell.getContext())}
-      </Cell>
-    ))}
-    {row.original.renderExpandedContent && <StyledExpandedContent row={row} />}
-    {row.subRows &&
-      row.subRows.map((subRow) => (
-        <Row
-          key={subRow.id}
-          className={css`
-            &[aria-hidden="false"] td > div[data-state="entered"] {
-              max-height: unset;
-            }
-          `}
-          row={subRow}
-          virtualRow={virtualRow}
-        >
-          {subRow.getVisibleCells().map((cell) => (
-            <Cell key={cell.id}>
-              {flexRender(cell.column.columnDef.cell, cell.getContext())}
-            </Cell>
-          ))}
-        </Row>
-      ))}
-  </Row>
+        data-cy={dataCyRow}
+        data-index={row.index}
+        data-selected={isSelected}
+        row={row}
+        virtualRow={virtualRow}
+      >
+        {row.getVisibleCells().map((cell) => (
+          <Cell
+            key={cell.id}
+            cell={cell}
+            className={cellStyle}
+            style={cellPaddingStyle}
+          >
+            {flexRender(cell.column.columnDef.cell, cell.getContext())}
+          </Cell>
+        ))}
+      </Row>
+    )}
+    {row.isExpandedContent && <StyledExpandedContent row={row} />}
```

Comment:
`StyledExpandedContent` is showing a TypeError in my editor. 
<img width="1115" alt="Screenshot 2025-04-18 at 12 18 15 PM" src="https://github.com/user-attachments/assets/51c697d2-793d-4144-b173-d451dad9a864" />

Reply:
I think its the same thing with `Typography` components where they throw errors but not compile errors 😞